### PR TITLE
README

### PR DESCRIPTION
**Pull request by hayleycd**
Start of README. I am going to add a bunch of comments and questions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,80 @@
...
+
+![Image indicates the location of the security actions section and the "view all" link.](./images/security-actions.png)
+
+3) Search for "OSV-Scanner". 
+
+![Image shows the GitHub Actions search bar.](./images/actions-tab.png)
+
+4) Choose the "OSV-Scanner" from the list of workflows, and then click “Configure”.
+
+TODO: Insert image
```
Comment (hayleycd) [inline README.md:46]: I can't get this image until it is released.

File: README.md [hunk 2]
```diff
@@ -1,8 +1,80 @@
...
+
+5) Commit the changes.
+
+TODO: Insert image
```
Comment (hayleycd) [inline README.md:50]: I can't get this image until it is released.

File: README.md [hunk 3]
```diff
@@ -1,8 +1,80 @@
...
+[OSV-Scanner](https://google.github.io/osv-scanner) provides an officially supported frontend to the [OSV database](https://osv.dev) that connects a project’s list of dependencies with the vulnerabilities that affect them. Since the OSV database is open source and distributed, it has several benefits in comparison with closed source advisory databases and scanners:
+
+- Each advisory comes from an open and authoritative source (e.g. the [RustSec Advisory Database](https://github.com/rustsec/advisory-db))
+- Anyone can suggest improvements to advisories, resulting in a very high quality database
+- The OSV format unambiguously stores information about affected versions in a machine-readable format that precisely maps onto a developer’s list of packages
+
+
+## Installation
+
+OSV-Scanner has a primary GitHub action that scans the repository and can be configured to scan on a schedule or on release. OSV-Scanner also has a secondary GitHub Action that performs a scan on pull request and reports newly introduced vulnerabilities.
```
Comment (hayleycd) [inline README.md]: Is this true? I believe so based on the information you sent me, but I want to be sure this is looking correct. @another-rex

File: README.md [hunk 4]
```diff
@@ -1,8 +1,80 @@
...
+
+OSV-Scanner has a primary GitHub action that scans the repository and can be configured to scan on a schedule or on release. OSV-Scanner also has a secondary GitHub Action that performs a scan on pull request and reports newly introduced vulnerabilities. 
+
+The primary GitHub Action can be installed [automatically](#automatic-installation) or [manually](#manual-installation). 
+
+The secondary GitHub Action that runs on pull request must be installed [manually](#manual-installation)
+
+### Automatic installation
+
+1) From your GitHub project's main page, click “Actions” tab in the navigation bar.
```
Comment (hayleycd) [inline README.md:32]: The steps here deviate from what is shown on Scorecard Action, but those steps are stale now. This has fewer clicks and is fresher.

File: README.md [hunk 5]
```diff
@@ -1,8 +1,75 @@
...
+
+## Scheduled scan
+Regularly scanning your project for vulnerabilities can alert you to new vulnerabilities in your dependency tree. The scheduled scan will scan your project on a set schedule and report all known vulnerabilities. If vulnerabilities are found the action will return a failed status.
+
+## Installation
+
+The OSV-Scanner GitHub Action can be [automatically](#automatic-installation) or [manually](#manual-installation) installed. 
+
+### Automatic installation
+
```
Comment (hayleycd) [inline README.md:31]:
```suggestion
```

File: README.md [hunk 6]
```diff
@@ -1,8 +1,74 @@
...
+To manually install the GitHub Action, please follow instructions on our [main documentation page](https://google.github.io/osv-scanner/github-action/).
+
+## Customization
+
+To learn more about optional inputs for the GitHub Action, please see our [main documentation page](https://google.github.io/osv-scanner/github-action/#customization).
+
+## View results
+
+Maintainers can review results of scheduled scans by navigating to their project's `security > code scanning` tab. Vulnerability details can also be viewed by clicking on the details of the failed action.
```
Comment (another-rex) [inline README.md]:
```suggestion
Maintainers can review results of scheduled scans by navigating to their project's `Security > Code Scanning` tab. Vulnerability details can also be viewed by clicking on the details of the failed action.
```

**Review by another-rex — APPROVED** [review]
Looks great!

**Pull request merged**